app-emulation/spice-vdagent/spice-vdagent-0.22.1.ebuild

```
@@ -0,0 +1,67 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit linux-info tmpfiles
+
+DESCRIPTION="SPICE VD Linux Guest Agent"
+HOMEPAGE="https://www.spice-space.org/"
+SRC_URI="https://www.spice-space.org/download/releases/${P}.tar.bz2"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gtk selinux systemd"
+
+CDEPEND="
+	dev-libs/glib:2
+	>=app-emulation/spice-protocol-0.14.0
+	media-libs/alsa-lib
+	sys-apps/dbus
+	>=x11-libs/libpciaccess-0.10
+	x11-libs/libdrm
+	x11-libs/libXfixes
+	x11-libs/libXrandr
+	x11-libs/libX11
+	x11-libs/libXinerama
+	gtk? ( x11-libs/gtk+:3 )
+	systemd? ( sys-apps/systemd )"
+DEPEND="${CDEPEND}
+	virtual/pkgconfig"
+RDEPEND="${CDEPEND}
+	selinux? ( sec-policy/selinux-vdagent )"
```

```suggestion
DEPEND="
	dev-libs/glib:2
	>=app-emulation/spice-protocol-0.14.0
	media-libs/alsa-lib
	sys-apps/dbus
	x11-libs/libdrm
	>=x11-libs/libpciaccess-0.10
	x11-libs/libXfixes
	x11-libs/libXrandr
	x11-libs/libX11
	x11-libs/libXinerama
	gtk? ( x11-libs/gtk+:3 )
	systemd? ( sys-apps/systemd )"
RDEPEND="${DEPEND}
	selinux? ( sec-policy/selinux-vdagent )"
BDEPEND="virtual/pkgconfig"
```

`pkgconfig` is a BDEPEND and therefore we can delete the "COMMON_DEPEND" condition here.

---

I see, I think I have made the appropriate changes recommended now.